src/deps_info.json

```diff
@@ -93,7 +93,6 @@
   "emscripten_websocket_new": ["malloc"],
   "emscripten_wget_data": ["malloc"],
   "emscripten_webgl_destroy_context": ["emscripten_webgl_make_context_current", "emscripten_webgl_get_current_context"],
-  "emscripten_webgl_create_context": ["malloc", "free"],
```

Comment:
Isn't this still needed in the pthread case?

Reply:
Oh, the pthreads runtime is guaranteed to bring in malloc/free anyhow (it uses it for proxying and other things). I'd add a comment in the JSON file but it won't parse with comments (not valid JSON).

If we had a mechanism in this file to say "only in a certain mode" then we would make this line be only in pthreads, for future use I guess. But even if in the far future this ends up causing breakage, these aren't surprising bugs - we'd see an unresolved symbol error in our tests, so I'm not worried here.